.github/workflows/airflow-publish.yml

```diff
@@ -25,52 +25,37 @@ on:
       release-config:
         description: "Path to the release config file"
         required: true
-        default: "release-config.yml"
+        default: "airflow-rc-config.yml"
+        type: choice
+        options:
```

Comment:
Added choice option so it can be selected , single workflow but two separate configs, one for rc and another final release pypi (or may some better name? :) )

Reply:
Good.